static/app/views/teamInsights/projectStabilityColumn.tsx

```diff
@@ -0,0 +1,209 @@
+import styled from '@emotion/styled';
+import round from 'lodash/round';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import {DateTimeObject, getDiffInMinutes} from 'app/components/charts/utils';
+import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
+import Placeholder from 'app/components/placeholder';
+import {DEFAULT_STATS_PERIOD} from 'app/constants';
+import {IconArrow} from 'app/icons';
+import {tct} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project, SessionApiResponse} from 'app/types';
+import {defined, percent} from 'app/utils';
+import {formatAbbreviatedNumber} from 'app/utils/formatters';
+import {getPeriod} from 'app/utils/getPeriod';
+import {Color} from 'app/utils/theme';
+import {shouldFetchPreviousPeriod} from 'app/views/projectDetail/utils';
+import {displayCrashFreePercent, getCrashFreePercent} from 'app/views/releases/utils';
+
+type Props = AsyncComponent['props'] & {
+  project: Project;
+  organization: Organization;
+  hasSessions: boolean | null;
+} & DateTimeObject;
+
+type State = AsyncComponent['state'] & {
+  currentSessions: SessionApiResponse | null;
+  previousSessions: SessionApiResponse | null;
+};
+
+class ProjectStabilityColumn extends AsyncComponent<Props, State> {
+  shouldRenderBadRequests = true;
+
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      currentSessions: null,
+      previousSessions: null,
+    };
+  }
+
+  getEndpoints() {
```

Comment:
Does this mean we'll make api call for every single project?
We could instead either:
- batch projects into single session API call (you can provide an array of projects and then group by them)
- call /projects endpoint with `&sessionStats=1` and you'll get crash free rate for this and previous period already calculated from backend (see projects page `/organizations/sentry/projects/`)

Reply:
Yeah it would be two requests per project. I don't know if `/projects` is an option since it doesn't take a start/end.

I can try providing an array of projects and grouping